bcrypt.js

```diff
@@ -36,6 +36,10 @@ module.exports.genSalt = function(rounds, ignore, cb) {
         cb = arguments[1];
     }
 
+    if(!cb) {
```

Comment:
style nitpick: space after `if` (basically, look at the rest of the file and try to follow that style for any new code you add.